pkg/waveai/openaibackend.go

```
@@ -59,19 +59,19 @@ func convertPrompt(prompt []wshrpc.OpenAIPromptMessageType) []openaiapi.ChatComp
 	return rtn
 }
 
-func convertUsage(resp openaiapi.ChatCompletionResponse) *wshrpc.OpenAIUsageType {
+func convertUsage(resp openaiapi.ChatCompletionResponse) *wshrpc.WaveAIUsageType {
```

golangci-lint (1.62.2)

func `convertUsage` is unused

(unused)